frontend/src/metabase/visualizations/lib/renderer_utils.js

```diff
@@ -79,7 +79,9 @@ export function getXValues({ settings, series }) {
         isDescending = isDescending && value <= lastValue;
       }
       lastValue = value;
-      uniqueValues.add(value);
+      if (value != null) {
```

Comment:
This is what we already do in some other place, so there is no regressions https://github.com/metabase/metabase/blob/4ec1fd883f5ac249ac7dac3e17986105bdac5e5c/frontend/src/metabase/visualizations/echarts/cartesian/utils/timeseries.ts#L95. Unfortunately, `computeTimeseriesDataInverval` is called with raw data as well so we cannot remove it from there without some refactoring.